wfe/web-front-end_test.go

```diff
@@ -76,8 +74,29 @@ eROL1ve1vmQF3kjrMPhhK2kr6qdWnTE5XlPllVSZFQenSTzj98AO
 		"n":"m5Cpx3vZ0CjATirDpbILvq78fm3Dv5RBkO1VLWFmJj5Mb54vc9oYZWc1V1k-LJoESuuPHhaNO2Eu8T9tslQWcZSzr5NImxAwMk970gVQa-Hqv-Jr6xstrBpq7TKpXHTx2FnfA2wQrfIQSlBXu0t4jdUOr3oJh-QXvma8nLITdtjpC0AZNtqd0QkRJX_90SaNrl18Rr_0JrBH9ZmUSFcf3mo_BtL0Gx0jE3n-iwCI8rQtfyVP__9-n__r4IhalKLzaeio6o-qrdemh0EZgjKGCS1_RpTIeArkO8uia1KgOq-z-GfemKEm4s07WO_a0_9dLqbvpnyyZvUi405m3vGDfQ",
 		"e":"AAEAAQ"
 	}`
+
+	RandomCert = "308201ba30820166a00302010202020675300b06092a864886f70d01010b3034" +
+		"3110300e06035504061307417573747269613111300f060355040a1308486561" +
+		"706c6f636b310d300b060355040b130454484f52301e170d3039313131303233" +
+		"303030305a170d3139313131303233303030305a30343110300e060355040613" +
+		"07417573747269613111300f060355040a1308486561706c6f636b310d300b06" +
+		"0355040b130454484f52305c300d06092a864886f70d0101010500034b003048" +
+		"024100a4a5007fbcda762d88f72ac6601571d989b397a7c2fbc260247c773e0b" +
+		"95c09d100b43170fe06c4a649529078b90f9e611a89ca470b3513f49fbb42e55" +
+		"5c6cd90203010001a3643062300e0603551d0f0101ff040403020084301d0603" +
+		"551d250416301406082b0601050507030206082b06010505070301300f060355" +
+		"1d130101ff040530030101ff300e0603551d0e04070405010203040530100603" +
+		"551d230409300780050102030405300b06092a864886f70d01010b034100212f" +
+		"9cadea729d8eb36d9dfc035f2f16bd561a0b30dfa5f838c0cc71289ee5faeed6" +
+		"0d5c3eb1a87aecac827855dcc2fb89ddbfe05ebcdc667d0f2e31ccf1d90d"
 )
 
+var wfe = NewWebFrontEndImpl()
```

Comment:
Why is this defined at the top level, as opposed to within a test?  That's how we got into trouble with dependent tests in the RA tests.